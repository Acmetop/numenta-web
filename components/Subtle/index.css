/**
 * Numenta shared web components
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.subtle {
  composes: silver from 'tachyons';
}
