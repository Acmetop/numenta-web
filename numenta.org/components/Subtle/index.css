/**
 * Numenta.org HTM Community website source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.subtle {
  composes: silver from 'tachyons';
}
