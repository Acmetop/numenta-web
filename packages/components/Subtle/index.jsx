// Numenta shared web components
// MIT License (see LICENSE.txt)
// Copyright © 2005—2016 Numenta <http://numenta.com>

import React from 'react'

import styles from './index.css'


/**
 *
 */
const Subtle = ({children}) => (
  <span className={styles.subtle}>
    {children}
  </span>
)

Subtle.propTypes = {
  children: React.PropTypes.node.isRequired,
}

export default Subtle
