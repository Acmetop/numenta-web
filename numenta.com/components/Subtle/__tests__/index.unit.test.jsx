import React from 'react'
import renderer from 'react-test-renderer'

import Subtle from '../../Subtle'


describe('<Subtle /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <Subtle>Quiet Text</Subtle>
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
