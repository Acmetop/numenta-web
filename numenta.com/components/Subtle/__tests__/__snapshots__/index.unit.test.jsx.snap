exports[`<Subtle /> React component unit test suite Renders correctly 1`] = `
<span
  className="subtle">
  Quiet Text
</span>
`;
